Tap: disable tests being built for Releases

shymega opened the pull request:
As per @vancluever's Q on the Input Leap Discussion thread (see: input-leap/input-leap#1678), this commit disables the tests from being built.

vancluever commented:
Thanks @shymega! Will take a look here sometime today!

shymega commented:
Np. Maybe we could trigger CI in the meantime?

vancluever commented:
@shymega done! Just a note though I don't think I've set up any meaningful tests for this yet so the CI might not be useful.

shymega commented:
@vancluever Well, I just wanted to see if it builds. That's all I need to see to be honest.

shymega committed:
Tap: disable tests being built for Releases

As per @vancluever's Q on the Input Leap Discussion thread (see:
input-leap/input-leap#1678), this commit disables the tests from being
built.

Signed-off-by: Dom Rodriguez <shymega@shymega.org.uk>

shymega commented:
Fixed in latest commit. Syntax typo on my part.

vancluever commented:
Yeah was just going to say. I just know I took out some stuff that was kind of required if you wanted to make the tap a part of homebrew-core, and haven't really looked at it since, so I wasn't too sure if it was going to do the full build/install. But at least it caught a syntax error! 🙂 Watching the CI too otherwise.

shymega commented:
Yeah, I think it's best we keep the Tap separate for now. Once Input Leap is stable, we can release on core.

Let's see what the latest checks do.

vancluever commented:
@shymega not too sure what the build error is about given the amount progress it makes (almost looks like it finishes in the logs but then it reports ninja failing a subcommand).  I've just merged it anyway and it still builds fine over here.

This is curious, not too sure what's up here, if there's missing logs or what or if `build_dist.sh` returned non-zero after success for some reason. Can investigate further if it's something that you think is worth following!

```
   [219/219] cd /tmp/input-leap-20230815-8505-fbp5jv/build/bundle && bash build_dist.sh
  FAILED: CMakeFiles/InputLeap_MacOS /tmp/input-leap-20230815-8505-fbp5jv/build/CMakeFiles/InputLeap_MacOS 
  cd /tmp/input-leap-20230815-8505-fbp5jv/build/bundle && bash build_dist.sh
  Checking for bundle contents
  Copying binaries into bundle
  Building app bundle
  Bundle created successfully
  ninja: build stopped: subcommand failed.
```
